frontend/src/app/libs/common-components/components/button/button.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent {
+
+	@Input()
+	public type: string = 'basic';
+
+	@Input()
+	public isDark: boolean = false;
+
+	@Input()
+	public isDisabled: boolean = false;
```

Comment:
lets not use `is`, just `disabled`, using `is` is good practice in js/ts, but in templates it looks kinda weird